Assets/Scripts/Helpers/IInteractable.cs

```diff
@@ -0,0 +1,9 @@
+using UnityEngine;
+
+namespace Helpers
+{
+    public interface IInteractable
+    {
+        void OnInteract(int interactable); // not sure what format will be passed so chose "int" for now
```

Comment:
It does not need a viariable passed in yet.

Reply:
yet...